### DISCUSSION TITLE
Example for next 'src' (not 'app') with twin@3.4.0

### DISCUSSION THREAD

**Discussion by drMonro** [ben-rogerson/twin.macro#823] [General]
'app' in Next still not stable (i think till Next@14), so can you please make example for 'src' directory?

Can't adjust with twin@3.4.0 and styled-components@6

Thank you

**Comment by ben-rogerson** [comment]
They are recommending it now as the way to use Next, so it can't be unstable still - right?
I can't say the same on the implementation of RSC - but that's another topic. 

![image](https://github.com/ben-rogerson/twin.macro/assets/21288568/f8b7b4e2-8e61-42c8-8a93-8190a36a8c4a)

Could you let me know a little more about what you mean by:

> Can't adjust with twin@3.4.0 and styled-components@6

It's possible to tweak the template/setup by changing the paths at the top of the `withTwin.js` file.

**Reply by ben-rogerson** [reply]
Just saw this was closed - so you can ignore :)
